### PR TITLE
Move collector polling into a go routine

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

File: pkg/runner/runner.go [hunk 1]
```diff
@@ -17,6 +19,11 @@ import (
 	"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/utils"
 )
 
+const (
+	maxConnsecutivePollErrors = 3
```
Comment (crwr45) [inline pkg/runner/runner.go:23]: This should be MUCH higher, or even never cause collection to stop. If we are collecting data over days, or power cycling machines, or any number of other things, the collector should include failure to collect in the output but not halt.

**Review by crwr45 — CHANGES_REQUESTED** [review]

File: pkg/runner/runner.go (hunk 1 above)
Comment (nocturnalastro) [reply pkg/runner/runner.go:23]: Fair. Perhaps this is something that should be configurable.

Comment (nocturnalastro) [reply pkg/runner/runner.go:23]: For now I'll set it to 1800 which is 30 mins at a rate of 1 per second.

**Comment by nocturnalastro** [comment]
Superseded by #30

**Pull request closed without merging**